openapi/multi-yaml/index.yaml

```diff
@@ -79,223 +79,141 @@ tags:
       | archived           | this sub account has been archived; they won't be able to process payments (but their record will remain for historical reasons) |
+
   - name: Platform Wallet Accounts
     description: |
-      Platform wallet accounts allow you to store payment methods centrally and use them across multiple sub-accounts within your platform. 
-      This feature enables platforms to maintain a single source of stored payment methods while processing payments through different sub-accounts.
+      A Platform Wallet Account allows you to store payment methods centrally and use them across multiple sub accounts within your platform.
+      This feature enables you to maintain a single source of stored payment methods while processing payments through different sub accounts.
+
+      ## Enable a Platform Wallet Account
+      <small>*Note: You can choose a sub account as your platform_wallet_account once it is underwritten and enabled for payments.*</small>
 
-      ### Enabling Platform Wallet Accounts
-      Contact JustiFi support to enable the `platform_wallet_account` setting for your designated wallet account.
+      Contact us at [customer_success@justifi.tech](mailto:customer_success@justifi.tech) to enable the `platform_wallet_account` setting for your designated platform wallet account. 
+
+      ## Key Features
       Once enabled, you can:
-      - Store payment methods in the designated wallet account
-      - Use these payment methods across your platform's sub-accounts
-      - Group payment methods for easier management. See [PaymentMethodGroups](http://localhost:3001/api-spec#tag/Payment-Method-Groups)
-
-      ### Using Platform Wallet Payment Methods
-      1. Create payment methods in your wallet account using the [Payment Methods API](http://localhost:3001/api-spec#tag/Payment-Methods/operation/CreatePaymentMethod):
-      ##### Example Request: Create Payment Method in Wallet Account
-      ```json
-      {
-        "payment_method": {
-          "card": {
-            "name": "Lindsay Whalen",
-            "number": "4242424242424242",
-            "verification": "123",
-            "month": 5,
-            "year": 2025,
-            "address_postal_code": "55555"
-          }
-        }
-      }
-      ```
+      - Store payment methods in the designated platform wallet account
+      - Use these payment methods across your platform's sub accounts
+      - Group payment methods for easier management using [PaymentMethodGroups](http://localhost:3001/api-spec#tag/Payment-Method-Groups)
+
+      ## Using Platform Wallet Payment Methods
+      <small>*Note: While the PaymentMethods/Payments API offers this functionality, we recommend using the [PaymentForm](https://storybook.justifi.ai/?path=/docs/payment-facilitation-payment-form--docs) for new integrations. (Available only to PCI-compliant platforms)*</small>
```

Comment:
I'd suggest changing this to: <small>*Note: While the PaymentMethods and Payments API allow you to tokenize a payment method we strongly suggest using the [Unified Fintech Checkout](https://storybook.justifi.ai/?path=/docs/payment-facilitation-unified-fintech-checkout%E2%84%A2--docs) or [PaymentForm](https://storybook.justifi.ai/?path=/docs/payment-facilitation-payment-form--docs) web components instead to avoid PCI scope.*</small>